native/cocos/platform/android/AndroidPlatform.cpp

```diff
@@ -222,10 +233,16 @@ class GameInputProxy {
             }
 
             for (int i = 0; i < motionEvent->pointerCount; i++) {
+                if (index >= 0) {
+                    i = index;
+                }
                 int id = motionEvent->pointers[i].id;
                 float x = GameActivityPointerAxes_getX(&motionEvent->pointers[i]);
                 float y = GameActivityPointerAxes_getY(&motionEvent->pointers[i]);
                 touchEvent.touches.emplace_back(x, y, id);
+                if (index >= 0) {
+                    break;
```

Comment:
So for pointer down and pointer up events, only one responsible touch will be added to the event, please confirm whether it's wanted behavior.

@dumganhar @PPpro